cloudflare/resource_cloudflare_worker_script.go

```diff
@@ -50,6 +52,20 @@ var secretTextBindingResource = &schema.Resource{
 	},
 }
 
+var webAssemblyBindingResource = &schema.Resource{
+	Schema: map[string]*schema.Schema{
+		"name": {
+			Type:     schema.TypeString,
+			Required: true,
+		},
+		"base64": {
+			Type:      schema.TypeString,
+			Required:  true,
+			Sensitive: true,
```

Comment:
What is the reasoning behind the base64 contents being marked as sensitive here?

Reply:
Woops, it shouldn't be, mistake when copy-pasting.